### ISSUE TITLE
Admin\Operator - Internal user can view Operators

### ISSUE THREAD

**Issue by zoeyli-46** [bcgov/cas-registration#1697]
#### Description:
As an internal user, I can see a list of all the operators 

#### Acceptance Criteria:

Given I am on the dashboard and click operators, or on the registration dashboard and click the operators tile
Then I am brought to the operators grid
[Figma](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?node-id=7861-1060&t=lP4wgjHuBjopAaSN-4) 

#### Development Checklist:

- [x] Create `bciers/apps/administration/app/components/operators/OperatorDataGrid.tsx`  similar to `bciers/apps/administration/app/components/operations/OperationDataGrid.tsx` 
- [x] In  `bciers/apps/administration/app/components/operators/OperatorDataGrid.tsx` include pagination similar to `bciers/apps/administration/app/components/operations/OperationDataGrid.tsx` 
- [x] Create `bciers/apps/administration/app/components/operators/OperatorDataGridPage.tsx` similar to `bciers/apps/administration/app/components/operations/OperationDataGridPage.tsx`
- [x] In `bciers/apps/administration/app/idir/cas_admin/operators/page.tsx` return `/operators/OperatorDataGridPage.tsx`` similar to `bciers/apps/administration/app/bceidbusiness/industry_user/operations/page.tsx`
- [x] In `bciers/apps/administration/app/idir/cas_analyst/operators/page.tsx` return `/operators/OperatorDataGridPage.tsx`` similar to `bciers/apps/administration/app/bceidbusiness/industry_user/operations/page.tsx`
- [x] Create `bciers/apps/administration/tests/components/operators/OperatorDataGridPage.test.tsx` similar to `bciers/apps/administration/tests/components/operations/OperationDataGridPage.test.tsx`
- [x] Create `bciers/apps/administration/tests/components/operators/OperatorDataGrid.test.tsx` similar to `bciers/apps/administration/tests/components/operations/OperationDataGrid.test.tsx`
- [ ] Meets the DOD

OR, because Brianna didn't realize Shon was already doing this:
- [ ] ~we can reuse the endpoint from bc_obps/registration/api/v1/user_operators.py, so no backend work in this card~
- [ ] ~create InternalUserOperatorsDataGridLayout, UserOperatorDataGridPage, UserOperatorDataGrid components in reg 2 (copy/paste over from reg 1 if they exist and then tweak to match wireframes)~
- [x] in bciers/apps/administration/app/idir/cas_admin/operators/page.tsx (and the same cas_analyst file), add the new components (see /app/bceidbusiness/industry_user_admin/operations/page.tsx for an example)
- [x] update the dashboard tile links to point to the new page route (no migration needed, just reset the db to see locally)
- [x] write vitests (see the operations component tests for an example of what to test where)
- [x] Checklist item
- [ ] Meets the DOD

Also:

- [x] Create operator endpoint, service, etc...
- [x] Create pytests for endpoints

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [x] User story is included
- [x] User role and type are identified
- [x] Acceptance criteria are included
- [x] Wireframes are included (if required)
- [x] Design / Solution is accepted by Product Owner
- [x] Dependencies are identified (technical, business, regulatory/policy)
- [x] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [x] Acceptance criteria are tested by the CI pipeline
- [x] UI meets accessibility requirements
- [x] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [x] Can be demoed in Sprint Review
- [x] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Dependencies

- Blocked by
- Blocking
